yt_dlp/extractor/viu.py

```diff
@@ -17,6 +16,7 @@
     smuggle_url,
     unsmuggle_url,
     url_or_none,
+    compat_urlparse,
```

Comment:
use urllib.parse

Reply:
Yeah that was I used first and thinking I'm doing wrong and not following the guide =D